test/libyul/yulSSAControlFlowGraph/function.yul

```diff
@@ -29,9 +29,8 @@
 // Block0_0 [label="\
 // Block 0\nv0, v1 := i()\l\
 // h(v0)\l\
-// h(v1)\l\
 // "];
-// Block0_0Exit [label="MainExit"];
+// Block0_0Exit [label="Terminated"];
```

Comment:
I am not 100% sure about this right now - `h` indeed cannot continue and terminates after call, does that mean that the main exit should be a `Terminated` as well? Does it matter at all?

Reply:
Comparing with the non-SSA CFG, the Block0Exit is also Terminated so it is consistent like this

Reply:
Looks good as is!